edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/requests/ModuleLogsRequestHandler.cs

```diff
@@ -108,10 +112,10 @@ public static void MismatchedMinorVersions(string payloadSchemaVersion, Version
                 Log.LogWarning((int)EventIds.MismatchedMinorVersions, $"Logs upload request schema version {payloadSchemaVersion} does not match expected schema version {expectedSchemaVersion}. Some settings may not be supported.");
             }
 
-            public static string LargePayload(int size, ModuleLogOptions options)
+            public static string LargePayload(int size, IEnumerable<ModuleLogOptions> options)
             {
                 // TODO: make/get aka link for documentation
-                string message = $"The payload is too large for a direct method. {Name} supports up to {MaxPayloadSize} bytes of logs. The current request returned {size} bytes.\nTry reducing the size of the logs by setting the 'tail', 'since' and 'from' fields in log options filter. For more information, see [[aka.ms/TODO:LinkToDocumentation]].\nCurrent options settings are:\n{Newtonsoft.Json.JsonConvert.SerializeObject(options, Newtonsoft.Json.Formatting.Indented)}";
+                string message = $"The payload is too large for a direct method. {Name} supports up to {MaxPayloadSize} bytes of logs. The current request returned {size} bytes.\nTry reducing the size of the logs by setting the 'tail', 'since' and 'from' fields in log options filter. For more information, see https://aka.ms/iotedge-log-pull \nCurrent options settings are:\n{Newtonsoft.Json.JsonConvert.SerializeObject(options, Newtonsoft.Json.Formatting.Indented)}";
```

Comment:
Is there _any_ possibility of this message exceeding the max payload size? 😃

Reply:
extremely unlikely. The options sent have to be under the limit, so the options would have to be within a hundred chars of the max size. I'm not concerned.